minemeld/flask/statusapi.py

```diff
@@ -152,8 +151,9 @@ def get_system_status():
     res['memory'] = psutil.virtual_memory().percent
     res['swap'] = psutil.swap_memory().percent
     res['disk'] = psutil.disk_usage(data_path).percent
+    res['sns'] = SNS_AVAILABLE
```

Comment:
Instead of adding to the existing /status/system (as we agreed), could you create an additional endpoint called /status/info and return version and sns availability ?
This way we can directly show the engine version on the WebUI.

Thanks!
luigi

Reply:
OK